### ISSUE TITLE
Obj file open error (Malformed file: indexes go higher than the number of elements!)

### ISSUE THREAD

**Issue by KotaYonezawa** [CloudCompare/CloudCompare#1569]
**Describe the bug**

Thanks for great tool.
I faced obj file open error that is generated by blender obj file export.
I checked both 2.11.3 and 2.12.beta but the behavior was same.
This obj file can be successfully loaded using other tools like MeshLab and blender.
I'm not sure why CloudCompare failed to load it.

The log is like as follows:
```
[15:45:33] [OBJ] ***/error_cloudcompare.obj
[15:45:33] [OBJ] Failed to tesselate face
[15:45:33] [OBJ] 23 points, 29 faces
[15:45:33] [OBJ] Malformed file: indexes go higher than the number of elements! (v=23/tc=-1/n=-1)
[15:45:33] An error occurred while loading 'error_cloudcompare': malformed file
```

**How to reproduce**

Simply load attached obj file. (zipped to attach here.)
[error_cloudcompare.zip](https://github.com/CloudCompare/CloudCompare/files/7827064/error_cloudcompare.zip)

**Expected behaviour**

I think load should be completed successfully.

Following is load result using MeshLab.
![MeshLabOK](https://user-images.githubusercontent.com/39361859/148505951-6f387b71-ebd0-4227-a385-8db207e47769.jpg)

**Your environment**
 - CC Version (or git hash/tag): 2.11.3 and 2.12.beta
 - OS & Version: Windows 10

**Comment by dgirardeau** [comment]
Hi,

I can't reproduce the issue on my side. The OBJ file is loaded correctly with both versions, and it has 24 vertices.

In your case, it seems that CC fails to load one vertex. Probably the first one? I see that the object name is in japanese:
![image](https://user-images.githubusercontent.com/2735509/148639771-d8f2704b-8a40-4ed2-86d3-4749966c6bca.png)

I don't see why it would be an issue yet, but can you change it to something else (like 'test') and see if you still have a loading issue?

Thanks

**Comment by KotaYonezawa** [comment]
Hi
Thanks a lot for your comment!
Your suggestion is correct.

I replaced Japanese string to ascii characters using text editor, error was not occurred.

```
# Blender v2.92.0 OBJ File: '2G_studio_cryptomatte_normal_addoverlap.blend'
# www.blender.org
o Floor
v 4.700000 4.125000 0.000000
v 4.700000 -4.125000 0.000000
v 0.000000 -4.125000 0.000000
v -4.440000 -4.125000 0.000000
...
```
Maybe, this issue can be occurred by some locale dependent APIs used in CloudCompare?

Anyway, I'll be able to avoid this issue not to use Japanese character in blender file.
Thanks a lot!

(If this issue can be fixed, users who lives in non-English-speaking world will be happy.)

**Comment by dgirardeau** [comment]
Yes, I'll try to fix it, but sadly it doesn't happen on my system (with a French Windows OS).

I may try a blind fix, in which case I'll ask you to test it if that's ok with you?

**Comment by KotaYonezawa** [comment]
> I may try a blind fix, in which case I'll ask you to test it if that's ok with you?
Understood! Thanks a lot!

**Comment by dgirardeau** [comment]
Ok, it took me some time, but I believe I found the issue. Can you test with the latest 2.12.beta version online?

**Comment by KotaYonezawa** [comment]
Thanks a lot!!
I checked 2.12.beta (Windows, Archive version) to load obj file that I attached before.
[error_cloudcompare.zip](https://github.com/CloudCompare/CloudCompare/files/7827064/error_cloudcompare.zip)
And it can be successfully loaded using my PC.
![2 12 beta](https://user-images.githubusercontent.com/39361859/155041404-4648a8ad-055b-4e46-bd99-cc46dd74c8eb.jpg)

And I also checked that same obj load was not OK using latest stable release (2.11.3 Anoia) using same PC.
